usr/share/rear/backup/default/010_pre_backup_script.sh

```diff
@@ -3,3 +3,11 @@ if test "$PRE_BACKUP_SCRIPT" ; then
     AddExitTask "${POST_BACKUP_SCRIPT[@]}"
     eval "${PRE_BACKUP_SCRIPT[@]}"
 fi
+
+for command in "${PRE_BACKUP_COMMANDS[@]}"; do
+    Log "Running PRE_BACKUP_COMMANDS: '$command'"
+    AddExitTask "$command"
```

Comment:
this does not look correct. Note that the `PRE_BACKUP_SCRIPT` code adds `POST_BACKUP_SCRIPT` to exit tasks, not `PRE_BACKUP_SCRIPT` (presumably it does so to guarantee that `POST_BACKUP_SCRIPT` is executed even if the backup code aborts and never reaches the subsequent steps, although I don't see why it is inside the `if test "$PRE_BACKUP_SCRIPT"` block, because one should then do this even if `PRE_BACKUP_SCRIPT` is empty and `POST_BACKUP_SCRIPT` is not).

Reply:
You're right. That"s non sense